Tests/GitHubRepositories.tests.ps1

```diff
@@ -688,16 +850,167 @@ try
                 }
             }
 
-            AfterAll -ScriptBlock {
+            AfterAll {
                 Remove-GitHubRepository -Uri $repo.svn_url -Confirm:$false
             }
         }
     }
 
-    Describe 'Get repository languages' {
+    Describe 'GitHubRepositories\Set-GitHubRepositoryTopic' {
+        BeforeAll {
+            $repo = New-GitHubRepository -RepositoryName ([Guid]::NewGuid().Guid)
+            $topic = Set-GitHubRepositoryTopic -OwnerName $repo.owner.login -RepositoryName $repo.name -Name $defaultRepoTopic
+        }
 
-        Context -Name 'For getting repository languages' -Fixture {
-            BeforeAll -ScriptBlock {
+        Context -Name 'When setting a repository topic' {
+            It 'Should return an object of the correct type' {
+                $topic | Should -BeOfType PSCustomObject
+            }
+
+            It 'Should return the correct properties' {
+                $defaultRepoTopic | Should -BeIn $topic.names
+            }
+        }
+
+        Context -Name 'When clearing all repository topics' {
+            BeforeAll {
+                $topic = Set-GitHubRepositoryTopic -OwnerName $repo.owner.login -RepositoryName $repo.name -Clear
+            }
+
+            It 'Should return an object of the correct type' {
+                $topic.PSObject.TypeNames[0] | Should -Be 'GitHub.Topic'
+            }
+
+            It 'Should return the correct properties' {
+                $topic.names | Should -BeNullOrEmpty
+            }
+        }
+
+        AfterAll {
+            Remove-GitHubRepository -Uri $repo.svn_url -Confirm:$false
+        }
+    }
+
+    Describe 'GitHubRepositories\Get-GitHubRepositoryContributor' {
```

Comment:
I think you might have missed my overall comment from the previous PR review.   Some of these tests are duplicative of the ones that were introduced as part of #242.  You can see similar contributor tests in master right now:

https://github.com/microsoft/PowerShellForGitHub/blob/2385b5cf5d959a7581bf968f15f346d9a0ff816b/Tests/GitHubRepositories.tests.ps1#L751-L787

I recommend merging those together and removing duplicates (so, keeping the `AnonymousContributors` context, and possibly keeping the `$repo.owner.login | Should -BeIn $contributors.login` tests.